From c14c087c15222b6b494f47d898602c307202b85f Mon Sep 17 00:00:00 2001
From: Dang Huynh <danct12@disroot.org>
Date: Sun, 23 Jan 2022 03:08:39 +0700
Subject: [PATCH] data: Add PinePhone Pro feedbackd theme

This is a direct copy from pine64,pinephone.json, it works fine on
the Pro version.

The PinePhone Pro introduces a more powerful vibrator than it's
predecessor.

Signed-off-by: Dang Huynh <danct12@disroot.org>
---
 data/meson.build               |   1 +
 data/pine64,pinephone-pro.json | 175 +++++++++++++++++++++++++++++++++
 2 files changed, 176 insertions(+)
 create mode 100644 data/pine64,pinephone-pro.json

diff --git a/data/meson.build b/data/meson.build
index a806f97..54d7429 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -2,6 +2,7 @@ theme_json = [
   'oneplus,enchilada.json',
   'oneplus,fajita.json',
   'pine64,pinephone.json',
+  'pine64,pinephone-pro.json',
   'purism,librem5.json',
   'xiaomi,beryllium.json',
 ]
diff --git a/data/pine64,pinephone-pro.json b/data/pine64,pinephone-pro.json
new file mode 100644
index 0000000..2b62845
--- /dev/null
+++ b/data/pine64,pinephone-pro.json
@@ -0,0 +1,175 @@
+{
+  "name" : "default",
+  "profiles" : [
+    {
+      "name" : "full",
+      "feedbacks" : [
+        {
+          "event-name" : "alarm-clock-elapsed",
+          "type"       : "Sound",
+          "effect"     : "alarm-clock-elapsed"
+        },
+        {
+          "event-name" : "button-pressed",
+          "type"       : "Sound",
+          "effect"     : "button-pressed"
+        },
+        {
+          "event-name" : "button-released",
+          "type"       : "Sound",
+          "effect"     : "button-released"
+        },
+        {
+          "event-name" : "message-new-email",
+          "type"       : "Sound",
+          "effect"     : "message-new-email"
+        },
+        {
+          "event-name" : "message-new-instant",
+          "type"       : "Sound",
+          "effect"     : "message-new-instant"
+        },
+        {
+          "event-name" : "message-new-sms",
+          "type"       : "Sound",
+          "effect"     : "message-new-instant"
+        },
+        {
+          "event-name" : "phone-incoming-call",
+          "type"       : "Sound",
+          "effect"     : "phone-incoming-call"
+        },
+        {
+          "event-name" : "timeout-completed",
+          "type"       : "Sound",
+          "effect"     : "complete"
+        },
+        {
+          "event-name" : "theme-demo",
+          "type"       : "Sound",
+          "effect"     : "theme-demo"
+        },
+        {
+          "event-name" : "window-close",
+          "type"       : "Sound",
+          "effect"     : "theme-demo"
+        }
+      ]
+    },
+    {
+      "name" : "quiet",
+      "feedbacks" : [
+        {
+          "event-name"   : "alarm-clock-elapsed",
+          "type"         : "VibraPeriodic",
+          "magnitude"    : 32767,
+          "duration"     : 10000,
+          "fade-in-time" : 10000,
+          "fade-in-level": 4096
+        },
+        {
+          "event-name" : "bell-terminal",
+          "type"       : "VibraRumble",
+          "duration"   : 100
+        },
+        {
+          "event-name" : "button-pressed",
+          "type"       : "VibraRumble",
+          "duration"   : 80
+        },
+        {
+          "event-name" : "button-released",
+          "type"       : "VibraRumble",
+          "duration"   : 40
+        },
+        {
+          "event-name" : "message-new-email",
+          "type"       : "VibraRumble",
+          "duration"   : 750,
+          "count"      : 3,
+          "pause"      : 100
+        },
+        {
+          "event-name" : "message-new-instant",
+          "type"       : "VibraRumble",
+          "duration"   : 750
+        },
+        {
+          "event-name" : "message-new-sms",
+          "type"       : "VibraRumble",
+          "duration"   : 1250,
+          "count"      : 2,
+          "pause"      : 250
+        },
+        {
+          "event-name" : "message-sent-instant",
+          "type"       : "VibraRumble",
+          "duration"   : 250
+        },
+        {
+          "event-name" : "phone-incoming-call",
+          "type"       : "VibraPeriodic",
+          "duration"   : 10000
+        },
+        {
+          "event-name" : "timeout-completed",
+          "type"       : "VibraRumble",
+          "duration"   : 500
+        },
+        {
+          "event-name" : "theme-demo",
+          "type"       : "VibraPeriodic",
+          "magnitude"  : 32767,
+          "duration"   : 5000
+        },
+        {
+          "event-name" : "window-close",
+          "type"       : "VibraRumble",
+          "duration"   : 75
+        }
+      ]
+    },
+    {
+      "name" : "silent",
+      "feedbacks" : [
+        {
+          "event-name" : "battery-caution",
+          "type"       : "Led",
+          "color"      : "red",
+          "frequency"  : 5000
+        },
+        {
+          "event-name" : "phone-missed-call",
+          "type"       : "Led",
+          "color"      : "blue",
+          "frequency"  : 3000
+        },
+        {
+          "event-name" : "message-missed-email",
+          "type"       : "Led",
+          "color"      : "blue",
+          "frequency"  : 2000
+        },
+        {
+          "event-name" : "message-missed-instant",
+          "type"       : "Led",
+          "color"      : "blue",
+          "frequency"  : 1000
+        },
+        {
+          "event-name" : "message-missed-notification",
+          "type"       : "Led",
+          "color"      : "blue",
+          "frequency"  : 500
+        },
+        {
+          "event-name" : "message-missed-sms",
+          "type"       : "Led",
+          "color"      : "blue",
+          "frequency"  : 2000
+        }
+      ]
+    }
+  ]
+}
+
-- 
GitLab

